ibis/pandas/core.py

```diff
@@ -142,7 +126,9 @@ def is_computable_input_arg(arg):
 )
 
 
-def execute_with_scope(expr, scope, aggcontext=None, clients=None, **kwargs):
+def execute_with_scope(
```

Comment:
Can you add docs for `compute_localcontext` to the top of this file?

Reply:
Add docstring to both core.py and timecontext.py